### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Miking98**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,16 +65,17 @@ If you are using the STARR-OMOP dataset from Stanford (which uses the OMOP CDM),
 1. Download your STARR-OMOP dataset to `[PATH_TO_SOURCE_OMOP]`.
 2. Convert STARR-OMOP => MEDS using the following:
 ```bash
-femr_stanford_omop_fixer [PATH_TO_SOURCE_OMOP] [PATH_TO_SOURCE_OMOP]
-
 # Convert OMOP => MEDS data format
 meds_etl_omop [PATH_TO_SOURCE_OMOP] [PATH_TO_OUTPUT_MEDS]
```
Comment (EthanSteinberg) [inline README.md:69]: Let's do [PATH_TO_OUTPUT_MEDS]_raw

And then [PATH_TO_OUTPUT_MEDS] for the output of the fixer. Less chance of issues that way.

Comment (Miking98) [reply README.md]: Fixed! @EthanSteinberg

File: README.md [hunk 2]
```diff
@@ -65,16 +65,17 @@ If you are using the STARR-OMOP dataset from Stanford (which uses the OMOP CDM),
...
+
+# Apply Stanford fixes
+femr_stanford_omop_fixer [PATH_TO_OUTPUT_MEDS]_raw [PATH_TO_OUTPUT_MEDS]
 ```
 
 3. Use HuggingFace's Datasets library to load our dataset in Python
 ```bash
 import datasets
-dataset = datasets.Dataset.from_parquet(PATH_TO_OUTPUT_MEDS + 'data/*')
+dataset = datasets.Dataset.from_parquet([PATH_TO_OUTPUT_MEDS] + 'data/*')
```
Comment (EthanSteinberg) [inline README.md:78]: I think the brackets here [] will cause more confusion than clarity. Can you remove them?

File: README.md [hunk 3]
```diff
@@ -65,16 +65,17 @@ If you are using the STARR-OMOP dataset from Stanford (which uses the OMOP CDM),
...
 2. Convert STARR-OMOP => MEDS using the following:
 ```bash
-femr_stanford_omop_fixer [PATH_TO_SOURCE_OMOP] [PATH_TO_SOURCE_OMOP]
-
 # Convert OMOP => MEDS data format
-meds_etl_omop [PATH_TO_SOURCE_OMOP] [PATH_TO_OUTPUT_MEDS]
+meds_etl_omop [PATH_TO_SOURCE_OMOP] [PATH_TO_OUTPUT_MEDS]_raw
+
+# Apply Stanford fixes
+femr_stanford_omop_fixer [PATH_TO_OUTPUT_MEDS]_raw [PATH_TO_OUTPUT_MEDS]
```
Comment (EthanSteinberg) [inline README.md:72]: We probably want to make a comment here about --num_proc to get this to run at any reasonable speed.

File: README.md [hunk 4]
```diff
@@ -65,16 +65,17 @@ If you are using the STARR-OMOP dataset from Stanford (which uses the OMOP CDM),
 1. Download your STARR-OMOP dataset to `[PATH_TO_SOURCE_OMOP]`.
 2. Convert STARR-OMOP => MEDS using the following:
 ```bash
-femr_stanford_omop_fixer [PATH_TO_SOURCE_OMOP] [PATH_TO_SOURCE_OMOP]
-
 # Convert OMOP => MEDS data format
-meds_etl_omop [PATH_TO_SOURCE_OMOP] [PATH_TO_OUTPUT_MEDS]
+meds_etl_omop [PATH_TO_SOURCE_OMOP] [PATH_TO_OUTPUT_MEDS]_raw
```
Comment (EthanSteinberg) [inline README.md:69]: Likewise, needs a comment about num_shards and num_proc

File: README.md (hunk 2 above)
Comment (Miking98) [reply README.md:78]: fixed

File: README.md (hunk 4 above)
Comment (Miking98) [reply README.md:69]: I think this is still TBD pending the other MEDS ETL pull request so could we merge this readme fix first then open a new branch for this change?

Comment (EthanSteinberg) [reply README.md:69]: There are no MEDS ETL pull requests pending so I'm unsure what you mean by "TBD".

But that doesn't matter, feel free to merge this as is as we can add more comments later.

**Review by EthanSteinberg — APPROVED** [review]

**Pull request merged**